test/cpp/naming/generate_resolver_component_tests.bzl

```diff
@@ -90,13 +123,15 @@ def generate_resolver_component_tests():
                 "resolver_component_tests_runner_invoker.cc",
             ],
             external_deps = [
-                "absl/flags:flag",
-                "absl/log:check",
-                "absl/strings",
+                "@com_google_absl//absl/log:check",
```

Comment:
if we need to spell things out like this maybe it's time to eliminate `external_deps` entirely and just have copybara do the rewrites